src/ui/server/index.ts

```diff
@@ -122,6 +123,7 @@ ui.use('/insurance/eligibility', requiredInsuranceEligibilityDataProvided);
 ui.use('/insurance/:referenceNumber/*', getApplication);
 ui.use('/insurance/:referenceNumber/*', applicationAccess);
 ui.use('/insurance/:referenceNumber/*', applicationStatus);
+ui.use('/insurance/:referenceNumber/*', totalContractValueOverThreshold);
```

Comment:
Could we have this just after `getApplication`?

This new middleware could even be part of `getApplication` with a new mapping function that adds `totalContractValueOverThreshold` - not a blocker, but if we add another flag I think we should do this